docs/guides/how_to_guides/configuring_data_docs/how_to_host_and_share_data_docs_on_a_filesystem.rst

```diff
@@ -29,21 +29,46 @@ Steps
 
 2. **Test that your configuration is correct by building the site.**
 
-  Use the following CLI command: ``great_expectations docs build --site-name local_site``. If successful, the CLI will open your newly built Data Docs site and provide the path to the index page.
+.. content-tabs::
```

Comment:
This is awesome! The content tabs not being at the very top makes sense, but I wonder if there might be confusion about which API these docs are for? (This is less of an issue here, where the tabs are in the second step, and more of an issue with the GCS docs, where the tabs are a lot lower).

I guess I'm wondering if it's worth putting the content tabs at the top, and duplicating the code between tabs, or maybe putting a note at the top that the initial steps are the same, and when the paths diverge we'll have explanations? I don't love either of these, and might be overthinking, but wanted to raise.

Reply:
Great call! Done ✅